<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromoteMemeber</class>
 <widget class="QWidget" name="PromoteMemeber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PromoteMemeber</string>
  </property>
  <widget class="QLabel" name="cnic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;CNIC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PromoteMember">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Promote Member&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PromoteMemberPromote">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Promote</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="PromoteMemberName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="PromoteMemberCNIC">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Backbutton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>520</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="StatusBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cnic_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Status Bar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Backbutton</sender>
   <signal>clicked()</signal>
   <receiver>PromoteMemeber</receiver>
   <slot>Backbutton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PromoteMemberPromote</sender>
   <signal>clicked()</signal>
   <receiver>PromoteMemeber</receiver>
   <slot>promotemember()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Backbutton()</slot>
  <slot>promotemember()</slot>
 </slots>
</ui>
